src/Model/Job.php

```diff
@@ -36,6 +37,28 @@ public function getFullTitle(): string
         return implode(' ', $this->title);
     }
 
+    public function getFullRemuneration(): ?string
+    {
+        if (null !== $this->remunerationMin && null !== $this->remunerationMax) {
+            return sprintf('%s - %s K€', $this->remunerationMin / 1000, $this->remunerationMax / 1000);
+        }
+
+        if (null !== $this->remunerationMin && null === $this->remunerationMax) {
+            return sprintf('%s K€', $this->remunerationMin / 1000);
+        }
+
+        return null;
+    }
+
+    public function getExperienceInMonths(): ?string
+    {
+        if ($this->experience) {
```

Comment:
Pour fixer l'erreur de lint restante :

```suggestion
        if ($this->experience !== null) {
```

Reply:
My mistake, je me le note également pour tester `null`

Reply:
Merci ! 🙏